README.md

```diff
@@ -22,7 +22,7 @@ MailHog is an email testing tool for developers:
 
 1. Either:
   * [Download the latest release](/docs/RELEASES.md) of MailHog for your platform
-  * [Run it from Docker Hub](https://registry.hub.docker.com/u/mailhog/mailhog/) or using the provided [Dockerfile](Dockerfile)
+  * [Run it from Docker Hub](https://registry.hub.docker.com/r/mailhog/mailhog/) or using the provided [Dockerfile](Dockerfile)
```

Comment:
Even better to have https://hub.docker.com/r/mailhog/mailhog

Reply:
You are right. Corrected. Thanks